### PR TITLE
Add missing semicolon after statement

### PR DESCRIPTION
**Pull request by lusitdev**
#### Summary

Fixed missing semicolon in example code.
#### Motivation

To make the example code cleaner and less confusing.
#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/window/requestanimationframe/index.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ function step(timestamp) {
   }
 
   if (elapsed < 2000) { // Stop the animation after 2 seconds
-    previousTimeStamp = timestamp
+    previousTimeStamp = timestamp;
     !done && window.requestAnimationFrame(step);
```
Comment (teoli2003) [inline files/en-us/web/api/window/requestanimationframe/index.md:106]: Let's fix this at the same time (while technically correct, this is difficult to read)
```suggestion
    if (!done) {
      window.requestAnimationFrame(step);
    }
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

File: files/en-us/web/api/window/requestanimationframe/index.md (hunk 1 above)
Comment (lusitdev) [reply files/en-us/web/api/window/requestanimationframe/index.md:106]: Sure. I would write simplified. Is it okay like this?
```suggestion
    if (!done) window.requestAnimationFrame(step);
```

Comment (teoli2003) [reply files/en-us/web/api/window/requestanimationframe/index.md:106]: I prefer the long way as the braces prevent errors ( `if (!done) window.requestAnimationFrame(step); another_thing();` would not do the expected outcome. Always using braces prevents these bugs from creeping in when maintaining code.

Comment (lusitdev) [reply files/en-us/web/api/window/requestanimationframe/index.md]: Sorry, didn't see your comment. It's fixed now as per your suggestion.

**Review by teoli2003 — APPROVED** [review]
👍

**Pull request merged**